pkg/lib/bundle/validate.go

```diff
@@ -0,0 +1,256 @@
+package bundle
```

Comment:
Since `lib` is something the SDK team will be importing, isn't it better to keep all of these in one place? Moreover, If we are making changes that we do not want to affect the SDK team, functions can be isolated. What do you think? Thanks